examples/01_plotting/plot_demo_plotting.py

```diff
@@ -35,28 +38,50 @@
 localizer_tmap_filename = localizer_dataset.tmaps[1]
 
 ###############################################################################
-# Plotting statistical maps
+# Plotting statistical maps using function `plot_stat_map`
+
+# Import plotting module
+from nilearn import plotting
+
+# Visualizing t-map image on subject specific anatomical image with manual
+# positioning of coordinates using cut_coords given as a list
 plotting.plot_stat_map(localizer_tmap_filename, bg_img=localizer_anat_filename,
                        threshold=3, title="plot_stat_map",
-                       cut_coords=(36, -27, 66))
+                       cut_coords=[36, -27, 66])
 
 ###############################################################################
-# Plotting glass brain
+# Plotting statistical maps on glass brain image using function `plot_glass_brain`
+
+# Now, the t-map image is overlayed on glass brain image as fixed background
 plotting.plot_glass_brain(localizer_tmap_filename, title='plot_glass_brain',
                           threshold=3)
 
 ###############################################################################
-# Plotting anatomical maps
+# Plotting anatomical images using function `plot_anat`
+
+# Visualizing anatomical image of haxby dataset
 plotting.plot_anat(haxby_anat_filename, title="plot_anat")
 
 ###############################################################################
-# Plotting ROIs (here the mask)
+# Plotting ROIs (here the mask) using function `plot_roi`
+
+# Visualizing ventral temporal region image from haxby dataset overlayed on
+# subject specific anatomical image with coordinates positioned automatically on
+# region of interest (roi)
 plotting.plot_roi(haxby_mask_filename, bg_img=haxby_anat_filename,
                   title="plot_roi")
 
 ###############################################################################
-# Plotting EPI haxby
+# Plotting EPI image using function `plot_epi`
+
+# Import image processing tool
+from nilearn import image
+
+# Compute the mean functional image from haxby dataset in t dimension.
+# Basically reducing the functional image from 4D to 3D
```

Comment:
Alternative: "Compute the voxel-wise mean of activity maps across the time-series of maps"